Configuration/ProcessModifiers/python/jetCoreInPhase2_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.ProcessModifiers.splitClustersInPhase2Pixel_cff import splitClustersInPhase2Pixel
```

Comment:
why do you need this?

Reply:
I missed it while cleaning up. Line 3 now deleted.